x-pack/plugins/security_solution/public/resolver/view/panels/copyable_panel_field.tsx

```diff
@@ -9,39 +9,46 @@
 import { EuiToolTip, EuiPopover } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import styled from 'styled-components';
-import React, { memo, useState, useContext } from 'react';
+import React, { memo, useState } from 'react';
 import { WithCopyToClipboard } from '../../../common/lib/clipboard/with_copy_to_clipboard';
 import { useColors } from '../use_colors';
-import { ResolverPanelContext } from './panel_context';
+import { StyledPanel } from '../styles';
 
 interface StyledCopyableField {
   readonly backgroundColor: string;
   readonly activeBackgroundColor: string;
 }
 
 const StyledCopyableField = styled.div<StyledCopyableField>`
-  background-color: ${(props) => props.backgroundColor};
   border-radius: 3px;
   padding: 4px;
   transition: background 0.2s ease;
 
-  &:hover {
-    background-color: ${(props) => props.activeBackgroundColor};
-    color: #fff;
+  ${StyledPanel}:hover & {
+    background-color: ${(props) => props.backgroundColor};
+
+    &:hover {
```

Comment:
❔ Adding a comment to the hover just indicating that the keyboard accessibility is a known issue might save some time for someone in the future who is trying to figure that out.